src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -421,6 +421,7 @@ public Response getVersion(@Context ContainerRequestContext crc,
                                @PathParam("versionId") String versionId,
                                @QueryParam("excludeFiles") Boolean excludeFiles,
                                @QueryParam("includeDeaccessioned") boolean includeDeaccessioned,
+                               @QueryParam("returnOwners") boolean includeOwners,
```

Comment:
Minor detail: rename includeOwners to returnOwners to match the query param name as in other endpoints